Common/Market.cs

```diff
@@ -68,6 +69,8 @@ static Market()
         /// </summary>
         public const string Dukascopy = "dukascopy";
 
+        public const string Bitcoin = "bitcoin";
```

Comment:
I think the market he should be bitfinex, thoughts? This would make it analogous to forex where fxcm/oanda/dukascopy are all foreign exchange markets, much like bitfinex is a bitcoin market.